test/hotspot/jtreg/compiler/jvmci/TestEnableJVMCIProduct.java

```diff
@@ -67,24 +75,33 @@ public static void main(String[] args) throws Exception {
             new Expectation("EnableJVMCI", "false", "command line"),
             new Expectation("UseJVMCICompiler", "false", "default"));
         test("-XX:+EnableJVMCIProduct",
-            new Expectation("EnableJVMCIProduct", "true", "command line"),
+            new Expectation("EnableJVMCIProduct", "true", "(?:command line|jimage)"),
```

Comment:
What "(?:command line|jimage)" means? Also `|` hard to see.

Reply:
That's a standard Java regular expression and it means the value is expected have been set on the command line or by the VM options read from the jimage (see [JDK-8232080](https://bugs.openjdk.org/browse/JDK-8232080)).